rtsf-at-checkout-event-reconciler/events/process_events_test.go

```diff
@@ -30,159 +32,280 @@ type productInfoTest struct {
 
 func TestMain(m *testing.M) {
 
-	lc := logger.NewClient("app_functions_sdk_go", false, "./test.log", "DEBUG")
-	context = &appcontext.Context{
-		LoggingClient: lc,
-	}
-	m.Run()
-}
-
-func initCVReadingScannerENTER() models.Reading {
-	reading := models.Reading{
-		Name:  cvRoiEvent,
-		Value: `{"object_count":1,"product_name":"item 1","roi_action":"ENTERED","event_time":1559679684,"roi_name":"Scanner"}`,
-	}
-
-	return reading
-}
-
-func initCVReadingScannerEXIT() models.Reading {
-	reading := models.Reading{
-		Name:  cvRoiEvent,
-		Value: `{"object_count":1,"product_name":"item 1","roi_action":"EXITED","event_time":1559679695,"roi_name":"Scanner"}`,
-	}
+	lc := logger.NewMockClient()
+	context = pkg.NewAppFuncContextForTest("app_functions_sdk_go", lc)
 
-	return reading
+	m.Run()
```

Comment:
Standard way is to wrap m.Run in os.Exit

```suggestion
	os.Exit(m.Run())
```